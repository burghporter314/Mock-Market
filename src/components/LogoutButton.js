import React from 'react';
import { useAuth0 } from '@auth0/auth0-react';

const LogoutButton = () => {
    const { logout } = useAuth0();

    return(
        <div className="col-lg-4" style={{ "padding": "10px" }}>
            <input onClick={() => logout()} className="btn btn-lg btn-danger btn-block container-fluid" value="Log out" readOnly></input>
        </div>
    );
}

export default LogoutButton;