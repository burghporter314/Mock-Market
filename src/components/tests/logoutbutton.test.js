import React from 'react';
import LogoutButton from '../LogoutButton';
import renderer from 'react-test-renderer';

it('should match snapshot', () => {
    const component = renderer.create(
        <LogoutButton/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})