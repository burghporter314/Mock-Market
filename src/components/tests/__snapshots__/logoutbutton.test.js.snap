exports[`test should match snapshot 1`] = `
<button
  onClick={[Function]}>
  Log out
</button>
`;
